autoscale/src/main/java/com/sequenceiq/periscope/aspects/RequestLogging.java

```diff
@@ -11,10 +11,16 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(RequestLogging.class);
 
-    public <T> T logging(Supplier<T> callback, String requestName) {
+    public <T> T logResponse(Supplier<T> callback, String requestName) {
         long start = System.currentTimeMillis();
-        T o = callback.get();
-        LOGGER.debug("Ambari '{}' finished in {} ms", requestName, System.currentTimeMillis() - start);
-        return o;
+        T response = callback.get();
+        LOGGER.debug("Request '{}' finished in {} ms, Response {}", requestName, System.currentTimeMillis() - start, response);
```

Comment:
Updated to log only non sensitive information returned from toLoggableString. In interaction with other services this response metadata is required otherwise it is very difficult to identify production issues. Logs can be rotated and purged. I think kibana can handle log purge effectively. Also changed logging this to "info" from "debug" since this would be needed in production issues..

Reply:
INFO level for all YARN responses is far too much logging. With multiple clusters, and pulling data every 15odd seconds (when the windowing approach is used) - this will generate a ton of logging.
Can we summarize the results from YARN into a much shorter log message, if this is absolutely required for debugging? Otherwise debug level is better.

Reply:
yes i have over-riden tostring method of these dto to include absolute minimum fields.